### PR TITLE
issue-49 Changes wrt to Tags Block

### LINKED ISSUE #49: Tag Block Addition (News/Press Releases/Events Page/Healthy Thinking Page)
https://www.sunstar.com/healthy-thinking/how-has-covid-19-impacted-our-health/

Screenshot 2023-09-19 at 9 29 07 PM

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #49 

Test URLs:
- Before: https://main--sunstar--hlxsites.hlx.live/_drafts/piyush/healthy-thinking/how-has-covid-19-impacted-our-health
- After: https://issue-41--sunstar--hlxsites.hlx.live/_drafts/piyush/healthy-thinking/how-has-covid-19-impacted-our-health

- [ ]  If you are adding a new block or a variation to an existing block, has it been added in the [block-inventory](https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/Shared%20Documents/sites/sunstar/sunstar-engineering/_drafts/block-inventory.docx?d=w0e883b961c9f4401bae37da23ace83ec&csf=1&web=1&e=KcfAim) ?

### CODE REVIEW

**Comment by RitwikSrivastava** [comment]
Screenshot 2023-09-22 at 8 50 34 AM
@jindaliiita I can see a single tag repeated multiple times. Which is not the case on the sunstar.com page.

**Comment by jindaliiita** [comment]
> Screenshot 2023-09-22 at 8 50 34 AM @jindaliiita I can see a single tag repeated multiple times. Which is not the case on the sunstar.com page.

@RitwikSrivastava i have intentionally added random tags for my testing in draft document.

File: blocks/tags/tags.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+* decorates the tags block
+* @param {Element} block The social block element
+*/
+export default async function decorate(block) {
+  const tags = block.querySelectorAll('a');
+  block.innerHTML = '';
+
+  [...tags].forEach((tag) => {
```
Comment (sdmcraft) [inline blocks/tags/tags.js:9]: We already have the tags in metadata block. Could we use them? One difference though is that you need links for them. But it looks like you can build it from the tag name itself. If not, we could keep it in a spreadsheet.

Comment (jindaliiita) [reply blocks/tags/tags.js:9]: Yes, i can check that. Good point

**Comment by jindaliiita** [comment]
Tracked via https://github.com/hlxsites/sunstar/pull/60.
So, Closing this

**Pull request closed without merging**